packages/socket-mode/src/SocketModeClient.ts

```diff
@@ -420,7 +420,7 @@ export class SocketModeClient extends EventEmitter {
         const flatMessage = JSON.stringify(message);
         this.logger.debug(`Sending a WebSocket message: ${flatMessage}`);
         this.websocket.send(flatMessage, (error) => {
-          if (error !== undefined) {
+          if (error !== undefined && error !== null) {
```

Comment:
The following code also works. Either is fine.
```suggestion
          if (error) {
```